generated/nixnetsocket/nixnetsocket.proto

```diff
@@ -417,7 +417,7 @@ message InetNToAResponse {
 
 message InetNToPRequest {
   nidevice_grpc.Session stack_ref = 1;
-  Addr src = 2;
+  Addr addr = 2;
```

Comment:
Note: I'm always using "addr" for the various `Addr` encoded struct types (also the name used in nxsocket) but "address" for address strings (nxsocket also uses "address" for address strings, granted only in the IpStack API).